File: 20170120_Formatting_Strings/FormattingStrings.java
class FormattingStrings{
  public static void main(String[] args){
    // examples from https://docs.oracle.com/javase/tutorial/java/data/numberformat.html
    long n = 14505000;
    System.out.format("%d%n", n);
    System.out.format("%08d%n", n);
    System.out.format("%+8d%n", n);
    System.out.format("%,8d%n", n);
    System.out.format("%+,8d%n%n", n);
    }
}

File: 20170120_Formatting_Strings/FormattingStrings.py
class FormattingStrings():
    def printString(self):
        n = 14505000
        print(n)
        print("{:,}".format(n))
        print("+{:,}".format(n))



if __name__ == "__main__":
    fs = FormattingStrings()
    fs.printString()
